avocado/core/loader.py

```diff
@@ -658,22 +658,25 @@ def _find_avocado_tests(self, path, class_name=None):
                 docstring = ast.get_docstring(statement)
                 # Looking for a class that has in the docstring either
                 # ":avocado: enable" or ":avocado: disable
-                if (safeloader.check_docstring_directive(docstring, 'disable')
-                   and class_name is None):
+                has_disable = safeloader.check_docstring_directive(docstring,
```

Comment:
OK, so basically this blocks the manual check as @clebergnu setup requires this and we do not directly state it in our checkall script. Let's go with this fix and we'll investigate the difference in https://trello.com/c/f2OVNib1/1051-check-why-w503-is-not-being-caught-by-travis